CMakeLists.txt

```diff
@@ -100,16 +100,19 @@ else (USE_EXT_GD)
 	include(AC_HEADER_STDC)
 	include(CheckPrototypeExists)
 
+	if (ENABLE_GD_FORMATS)
+		FIND_PACKAGE(ZLIB)
```

Comment:
you want `REQUIRED` here (and below) so we abort if it's not found

we prob should be doing that for all FIND_PACKAGE calls actually (although you don't have to fix it here)

Reply:
yeah, we have another PR #472 to add `REQUIRED`.